collections-csharp/StackInCSharp/StackInCSharp/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using StackInCSharp;
+using System.Collections;
+
+List<string> letters = new List<string>() { "a", "b", "c" };
+Stack lettersStack = new Stack(letters);
+
+var page1 = new Page("The first visited page", DateTime.Now);
+var page2 = new Page("The second visited page", DateTime.Now);
+
+Console.WriteLine($"before push lettersStack contains {lettersStack.Count} items");
```

Comment:
You should have created a separate class with the wrapper methods for these methods demonstrated here in Program class. Then, you should have tested these methods and these collections. 

In the tests, you're creating new collections which means this code is untested.

Please refactor the code a bit to support this because this is how it should be done here and for the future articles.